discoursemap/modules/endpoint_module.py

```
@@ -373,36 +383,36 @@ def _test_endpoint(self, endpoint, category):
                     elif response.status_code in [401, 403]:
                         color = Fore.RED
                     else:
                         color = Fore.YELLOW
-                    
+
                     print(f"{color}[+] Found {category} endpoint: {endpoint} [{response.status_code}]{Style.RESET_ALL}")
-        
+
         except requests.exceptions.RequestException:
             pass  # Silently ignore connection errors
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)